src/main/groovy/edu/oregonstate/mist/mealcard/mapper/MealPlanBalanceMapper.groovy

```diff
@@ -0,0 +1,37 @@
+package edu.oregonstate.mist.mealcard.mapper
+
+import edu.oregonstate.mist.mealcard.core.MealPlanBalance
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+import java.time.ZoneId
+import java.time.ZonedDateTime
+import java.time.format.DateTimeFormatter
+
+class MealPlanBalanceMapper implements ResultSetMapper<MealPlanBalance> {
+
+    private static DateTimeFormatter dbFormatter = DateTimeFormatter
```

Comment:
Should these two be class level properties since they are only used in one method?